eclair-core/src/main/scala/fr/acinq/eclair/api/Service.scala

```diff
@@ -350,6 +361,7 @@ trait Service extends Logging {
     "channels: list existing local channels",
     "channels (nodeId): list existing local channels to a particular nodeId",
     "channel (channelId): retrieve detailed information about a given channel",
+    "channelstats: retrieves statistics about channel usage (fees, number and average amount of payments)",
```

Comment:
`audit` and `networkfees` are not documented, is it on purpose ?

Reply:
it was, but I changed my mind and they are now documented